src/types/custom.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module '*.svg' {
```

Comment:
Those `.d.ts` files are often located in root directory but I don't have strong opinion about it